flow/connectors/clickhouse/normalize.go

```diff
@@ -365,38 +376,51 @@ func (c *ClickHouseConnector) NormalizeRecords(
 				}
 			}
 
+			jsonExtractStringData := "JSONExtractString(_peerdb_data, '%s')"
```

Comment:
Should move to parametric format strings using `%[x]s`. Should be able to move it into some function that handles `enablePrimaryUpdate` too